### DISCUSSION TITLE
v5.0.1

### DISCUSSION THREAD

**Discussion by danharrin** [laravel-frontend-presets/tall#119] [Announcements]
## What's Changed
* Update Laravel Installation Link by @brandon-9 in https://github.com/laravel-frontend-presets/tall/pull/117
* Needs to be in array by @brandon-9 in https://github.com/laravel-frontend-presets/tall/pull/118

## New Contributors
* @brandon-9 made their first contribution in https://github.com/laravel-frontend-presets/tall/pull/117

**Full Changelog**: https://github.com/laravel-frontend-presets/tall/compare/v5.0.0...v5.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/laravel-frontend-presets/tall/releases/tag/v5.0.1'>v5.0.1</a>.</em>

**Comment by romaricvivien65** [comment]
Hi everybody.
Thanks for your work.
I have this in my terminal after install and vite running : [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.
Please help me to fix it !!!!

Best regards
